hw3/01-characters.html

```diff
@@ -6,14 +6,15 @@
     <title>01 - Characters</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <link rel="stylesheet" href="styles.css" />
     <script defer src="01-characters.js"></script>
   </head>
   <body>
     <main>
-      <h1>Exercise 01 - Characters</h1>
-      <section></section>
+      <h1 class="mt-5 mb-3">Exercise 01 - Characters</h1>
+      <section id="characters" class="mx-auto"></section>
```

Comment:
I think this should be `character-container` since it's more descriptive

also, since this is a list, I think it should be `ul` and the list item should be `li` instead of `div`